operations/helm/charts/mimir-distributed/CHANGELOG.md

```diff
@@ -28,6 +28,8 @@ Entries should include a reference to the Pull Request that introduced the chang
 
 ## main / unreleased
 
+* [BUGFIX] Do not set `securityContext` on Mimir components when `rbad.type` equals `scc`, since it conflicts with OpenShift. #4272
```

Comment:
```suggestion
* [BUGFIX] Do not set `securityContext` on Mimir components when `rbad.type` equals `scc` because it conflicts with OpenShift. #4272
```

Reply:
https://developers.google.com/style/word-list#letter-s